mozci/data/sources/activedata/__init__.py

```diff
@@ -68,6 +68,8 @@ def normalize(cls, result):
             }
             if item.get("result") in result_map:
                 item["result"] = result_map[item["result"]]
+            else:
+                logger.error(item)
```

Comment:
I guess this one was a debugging statement?

Reply:
Yes, removed.